### ISSUE TITLE
How to obtain training and test set?

### ISSUE THREAD

**Issue by atulapra** [isseu/emotion-recognition-neural-networks#17]
I have executed the `cvs_to_numpy.py` code after downloading `fer2013.csv` from [https://www.kaggle.com/c/challenges-in-representation-learning-facial-expression-recognition-challenge/data](Kaggle).

Now, it creates two files: 
`data_kike.npy` and labels_`kike.npy`.

From this, please tell me how to get the 4 files:
* data_set_fer2013.npy
* data_labels_fer2013.npy
* test_set_fer2013.npy
* test_labels_fer2013.npy

**Comment by denismakogon** [comment]
I assume validation data should be obtained here http://www.socsci.ru.nl:8180/RaFD2/RaFD.
But `data_kike.npy` and `labels_kike.npy` are `data_set_fer2013.npy` `data_labels_fer2013.npy`.

**Comment by NikAndrush** [comment]
@atulapra how you fixed your problem?

**Comment by atulapra** [comment]
@NikAndrush I did not train the model. I used pre-trained weights I found elsewhere.

**Comment by gaodlike** [comment]
@atulapra Does the pre-trained weights work well? It seems not very good in my computer

**Comment by denismakogon** [comment]
@gaodlike with my pretrained model efficiency is nearly 75-80%

**Comment by innagorda** [comment]
@atulapra can you send any link? I can't find(

**Comment by atulapra** [comment]
@innagorda You can find the model files in my repo [here.](https://github.com/atulapra/Emotion-detection)

**Comment by atulapra** [comment]
Issue solved.
